cpp/ps0005/p5Solution.cpp

```diff
@@ -1,32 +1,41 @@
 #include<iostream>
-#include<functional>
-using namespace std;
 
+using namespace std;
 
 int main() {
-	int num1;
-	int num2;
+	// prompting user for the 2 operands and an operator 1.5/2
+	double num1;
+	double num2;
 	char Operator;
 	cout << "Enter 2 numbers and a symbol of [+,-,*,/,%]: ";
 	cin >> num1 >> num2 >> Operator;

```

Comment:
- [x] declared variables to store the user input
- [ ] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input